Calibration/IsolatedParticles/src/FindDistCone.cc

```diff
@@ -79,9 +79,10 @@ namespace spr {
     else
       dR = 999999.;
     return dR;
-    if (debug)
+    /*    if (debug)
```

Comment:
Perhaps in this case one could revert the order of the return and debug statements, so that the info can still be printed if debug is required

Reply:
@perrotta , I did not want to change the behaviour thatis why I just commented it out. Currently the `if(){}` block is not executed so changing the order means it will be executed. If this is fine then sure I can change the order

Reply:
There are other `if(debug)` clauses in this code, I think this one was just inverted by mistake by the original author. Executing it looks to me a better approximation of the original intention, without adding to much load to the execution.
Alternatively, we could comment out all them (but this is outside the scope of your PR)

Reply:
ok, I will change the order